doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+.. _ug_nrf70_developing_raw_tx_operation:
+
+Raw packet transmission
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nrf70 series device supports the transmission of raw packets. Raw packets are packets that are not touched by the 802.11 MAC layer during transmission. The Application should provide an 802.11 conformant packet with transmit parameters which the 802.11 MAC layer will utilize to transmit packets.
```

Comment:
Pls, switch to using "nRF70 Series" in the right style :)
And we should refer to the API, here.

Proposal: The nRF70 Series device supports the transmission of raw IEEE 802.11 packets, via a dedicated raw TX API.

Reply:
"Raw packets are packets that are not touched by the 802.11 MAC layer during transmission"

This is ambiguous. Better explain briefly what be mean by "not touched". E.g.
"Raw packets are not modified by the 802.11 MAC layer on the nRF70 Series device".

Reply:
"The Application should provide an 802.11 conformant packet with transmit parameters which the 802.11 MAC layer will utilize to transmit packets."

From my discussion with @VivekUppunda I got the impression that the application _must_ provide (not should) , otherwise the MAC layer will drop the packets.

Proposal:
It is the responsibility of the application that uses the raw TX API to construct and provide fully-conformant 802.11 packets along with the required set of packet transmit and PHY parameters. The MAC layer on nRF70 Series device will transmit the raw packets on air using the provided packet  data, TX, and PHY parameters.

Reply:
And we probably need a link to the API reference for the "Raw TX API" :)

Reply:
okay

Reply:
Most of the above are addressed. There is no specific RAW TX API. The manner in which raw packets can be transmitted is to utilize the RAW socket support of Zephyr which allows a direct L2 layer transmission